modules/local/samplesheet_check.nf

```diff
@@ -0,0 +1,30 @@
+process SAMPLESHEET_CHECK {
+    tag "$samplesheet"
+
+    conda (params.enable_conda ? "conda-forge::python=3.9.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.9--1' :
+        'quay.io/biocontainers/python:3.9--1' }"
+
```

Comment:
Are they the same python version using conda or containers?

Reply:
Changed to python:3.8.3 as per the template.